src/java.base/share/classes/java/util/concurrent/Executors.java

```diff
@@ -435,7 +435,7 @@ public static Callable<Object> callable(Runnable task) {
      * @return a callable object
      * @throws NullPointerException if action null
      */
-    public static Callable<Object> callable(final PrivilegedAction<?> action) {
+    public static Callable<@Nullable Object> callable(final PrivilegedAction<?> action) {
```

Comment:
```suggestion
    public static Callable<@PolyNull Object> callable(final PrivilegedAction<@PolyNull ?> action) {
```

Reply:
Why didn't this method use a method type parameter to connect the argument to the result type?
Would using `@PolyNull` here express this correctly?